tools/build_utils.py

```diff
@@ -19,9 +18,7 @@
 import shlex
 import math
 import psutil as psu
-from sysconfig import get_paths
-from subprocess import check_output, call
-from wheel.vendored.packaging.tags import sys_tags
```

Comment:
Why were the from ... import ... statements removed?

Reply:
get_paths, check_output, sys_tags

the above modules are imported but not used.